2020-09-18 15:01:20,716 | 40 | common.Log | INFO | Log.py | info | 测试的日志输出INFO
2020-09-18 15:01:20,717 | 44 | common.Log | ERROR | Log.py | info | 测试的日志输出ERROR
2020-09-18 15:01:20,717 | 42 | common.Log | WARNING | Log.py | info | 测试的日志输出WARNING
2020-09-18 15:01:20,718 | 38 | common.Log | DEBUG | Log.py | info | 测试的日志输出DEBUG
2020-09-18 15:01:20,718 | 44 | common.Log | ERROR | Log.py | info | 测试的日志输出ERROR
2020-09-18 15:01:20,718 | 44 | common.Log | ERROR | Log.py | info | 测试的日志输出ERROR
2020-09-18 15:01:20,719 | 42 | common.Log | WARNING | Log.py | info | 测试的日志输出WARNING
2020-09-18 15:01:20,719 | 42 | common.Log | WARNING | Log.py | info | 测试的日志输出WARNING
2020-09-18 15:01:21,001 | 40 | common.Log | INFO | Log.py | info | test_getCase
2020-09-18 15:01:21,001 | 40 | common.Log | INFO | Log.py | info | test_getCase
2020-09-18 15:01:21,001 | 40 | common.Log | INFO | Log.py | info | test_getCase
2020-09-18 15:02:47,972 | 40 | common.Log | INFO | Log.py | info | test_getCase
2020-09-18 15:03:30,433 | 40 | common.Log | INFO | Log.py | info | test_getCase
2020-09-18 15:04:54,206 | 40 | common.Log | INFO | Log.py | info | test_getCase
2020-09-18 15:07:18,382 | 38 | common.Log | DEBUG | Log.py | debug | test_getCase
2020-09-18 15:08:32,032 | 38 | common.Log | DEBUG | Log.py | debug | test_getCase
2020-09-18 15:13:28,721 | 38 | common.Log | DEBUG | Log.py | debug | test_getCase
2020-09-18 15:16:25,697 | 40 | common.Log | INFO | Log.py | info | debug
2020-09-18 15:23:51,626 | 40 | common.Log | INFO | Log.py | info | debug
2020-09-18 15:23:51,627 | 54 | common.Log | DEBUG | Log.py | debug | test_getCase
2020-09-18 15:26:54,306 | 54 | common.Log | DEBUG | Log.py | debug | None
2020-09-18 15:27:53,136 | 54 | common.Log | DEBUG | Log.py | debug | None
2020-09-18 15:29:19,194 | 54 | common.Log | DEBUG | Log.py | debug | None
